lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,37 @@
 
 def palindrome_permutation?(string)
+  if string.empty?
+    return true
+  end
   
+  hash = {}
+  even_array = []
+  count = 0
+  odd_array = []
+  
+  str_array = string.split("")
+  
+  str_array.each do |letter|
```

Comment:
You can also iterate through a string with `string.each_char { |c| ` instead of splitting the string and creating a new array.